src/OpenTelemetry.Instrumentation.Process/README.md

```diff
@@ -0,0 +1,39 @@
+# Process Instrumentation for OpenTelemetry .NET
+
+[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.Process.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.Process.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)
+
+This is an [Instrumentation
+Library](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library),
+which instruments [.NET Process](https://docs.microsoft.com/dotnet) and
+collect telemetry about Process behavior.
+
+## Steps to enable OpenTelemetry.Instrumentation.Process
+
+### Step 1: Install package
+
+Add a reference to the
+[`OpenTelemetry.Instrumentation.Process`](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process)
+package.
+
+```shell
+dotnet add package OpenTelemetry.Instrumentation.Process
+```
+
+### Step 2: Enable Process instrumentation
+
+Process instrumentation should be enabled at application startup using the
+`AddProcessInstrumentation` extension on `MeterProviderBuilder`:
+
+```csharp
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddProcessInstrumentation()
+    .AddPrometheusExporter()
```

Comment:
nit: need to mention additional nuget installation for prometheus

Reply:
added installation info